src/pages/payments/cards/new/service/validations.ts

```diff
@@ -0,0 +1,104 @@
+import { FormType } from '../../type'
+import { CARD_HOLDER_NAME_MAX_LENGTH } from './const'
+
+const checkExpirationDate = (month: string, year: string) => {
+  const parsedMonth = parseInt(month)
+
+  if (!parsedMonth || !year) {
+    return false
+  }
+
+  if (parsedMonth < 1 || parsedMonth > 12) {
+    return false
+  }
+
+  return true
+}
+
+const checkCardholderName = (name: string) => {
+  if (name.length > CARD_HOLDER_NAME_MAX_LENGTH) return false
+
+  return true
+}
```

Comment:
이 부분은 취향이 갈릴 수도 있는데 아래처럼 작성할 수도 있겠네용 ㅎㅎ
```suggestion
const checkCardholderName = (name: string) => {
  return name.length <= CARD_HOLDER_NAME_MAX_LENGTH
}
```

Reply:
반영 했습니다!

[refactor(checkCardholderName): 불필요한 if문 제거](https://github.com/next-step/react-payments/pull/115/commits/7636189c7581ed766d069f7fbb13b2d286a9c724)